docs/Coiote_IoT_DM/firmware_update.md

```diff
@@ -78,28 +78,32 @@ The Firmware Update process is defined in the **Firmware Update Object `/5`**. T
 
     ![Basic Firmware Update](images/Basic Firmware Update.png)
 
+1. **Upload the firmware image**, which is usually a binary (`.bin`) file. 
+
+    !!! note 
+        When using the Anjay LwM2M Client, the binary file is called `app_update.bin`.
```

Comment:
This is not really specific to Anjay at all, but rather to the nRF Connect SDK. The same will be true for Zephyr LwM2M Client on those, while Anjay on e.g. STM32 boards will use a different file name.

Reply:
Ah, good to know. Will remove that part.